jetty-ee10/jetty-ee10-servlet/src/main/java/org/eclipse/jetty/ee10/servlet/ServletContextResponse.java

```diff
@@ -134,11 +134,17 @@ public ServletApiResponse getServletApiResponse()
 
     public void resetForForward()
     {
-        _httpOutput.resetBuffer();
-        _httpOutput.reopen();
+        _httpServletResponse.resetBuffer();
         _outputType = OutputType.NONE;
     }
 
+    public void included()
+    {
+        if (_outputType == OutputType.WRITER)
+            _writer.reopen();
+        _httpOutput.reopen();
```

Comment:
Why is this needed?  I thought the include response wrapper intercepts close, so it cannot be closed?
Is this for any sendErrors within an incliude?  A comment explaining would be good.

Reply:
I was just replicating all the places where this was called from Jetty 11. So maybe this is no longer needed. But then this will just be a NOOP anyway.

Reply:
Be wary of just copying behavours from jetty-11 as much as changed around things like dispatch and content management.  So good to question anything that looks suspect.
I'm guessing it is for sendErrors... but good to verify that it does do something sometimes and to then add a comment.